docs/pages/Getting_Started.md

```diff
@@ -2214,25 +2216,43 @@ the tests available.
 
 \[[top](#top)\]
 
-In this section we introduce how to add your observations to DART.
+First, you should understand that DART already supports a tremendous variety of
+observations. To fully support an observation means that the observation can be
+converted from its native format to the DART observation sequence format and that
+the observation forward operator is already implemented. Keep in mind that
+forward operators are not specific to any one model.
+
+The observation converters are in the *observations/obs_converter* directory
+and you should look there for the documentation describing which converters
+are available.
+
+The forward operators are functionally or logically grouped into Fortran
+modules in the *observations/forward_operator* directory. DART employs a 'contractual'
+style of programming in that the forward operator requests information from the
+model, and if the model cannot provide it, the forward operator may request different
+information in an attempt to collect the information needed to apply the operator.
+If the model cannot provide any of the required information, the forward operator fails,
+the DART QC for that observation is set to the appropriate value, and the program continues.
 
 1. [Background on DART observations](#obsBackground)
-2. [Simple observation definitions](#simpleObs)
-3. [Obs definitions for observations needing special handling](#complexObs)
-4. [Example observation definition](#exampleObsDef)
-5. [Observation sequence file](#obsSeqFile)
+1. [Adding new observation *quantities*](#newQUANTITIES)
+1. [Adding new observation *types*](#newTYPES)
+1. ['Simple' observation support](#simpleObs)
+1. [More complex observations](#complexObs)
+1. [Example observation definition](#exampleObsDef)
+1. [Observation sequence file](#obsSeqFile)
```

Comment:
Are these intended to all be "1"?  It looks okay in markup, but it's a bit jarring to see.

Reply:
I KNOW! Cool, n'est pas?   It beats renumbering when you insert a new item, so yes, it is deliberate. Too wierd? By having them all be the same, you can rearrange them at will without worrying about it.

Reply:
Yeah it's fine. I guess it is one of those Markdown things one just has to learn.